CSL-Traffic/Initializer.cs

```diff
@@ -1821,23 +1698,19 @@ public static void DumpTextureToPNG(Texture previewTexture, string filename = nu
                         Graphics.Blit(texture, rt);
                         DumpRenderTexture(rt, filename);
                         RenderTexture.ReleaseTemporary(rt);
-                        //Log.Warning(String.Format("Texture dumped to \"{0}\"", filename));
                     }
                     catch (Exception ex)
                     {
-                        //Log.Error("There was an error while dumping the texture - " + ex.Message);
```

Comment:
And here's another one.

What you want to do with them is up to you. It requires actual understanding of the code :D